fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/NetherBiomes.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.biomes.v1;
+
+import net.minecraft.world.biome.Biome;
+
+import net.fabricmc.fabric.impl.biome.NetherBiomesImpl;
+
+/**
+ * API that exposes the internals of Minecraft's nether biome code.
+ */
+public class NetherBiomes {
```

Comment:
Everywhere else in API (afaik) this class would be an interface with an INSTANCE field that NetherBiomesImpl implements (see e.g. net.fabricmc.fabric.api.client.model.ModelLoadingRegistry.) Why is this different here?

Not that it makes any real difference, but it would be more consistent.

Reply:
I wanted to be consistent with the rest of the biome API which follows the same pattern that I used.

Reply:
I wanted to be consistent with the rest of the biome API which follows the same pattern that I used.

Reply:
Hm, yeah I didn't even notice we did the OverworldBiomes one any differently. Also I noticed in OverworldBiomes we had the class as `final`, maybe put that here too?

Reply:
fixed 👍

Reply:
Yeah, if this style exists somewhere previously in API, then fair enough.